workflows/pypet-brian2/main.workflow

```diff
@@ -0,0 +1,31 @@
+workflow "pypet-brian2-example" {
+  resolves = "visualize"
+}
+
+action "install dependencies" {
+  uses = "jefftriplett/python-actions@master"
+  args = "pip install -r workflows/pypet-brian2/requirement.txt"
+  env = {
+    VENV_DIR="./workflows/pypet-brian2/venv"
```

Comment:
we can use the default value of `VENV_DIR` on this and the other actions to make it more straightforward